File: AntonAndPolyhedrons.java
import java.util.HashMap;
import java.util.Scanner;

public class AntonAndPolyhedrons {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        HashMap<String, Integer> polyhedronsList = new HashMap<>();
        polyhedronsList.put("Tetrahedron", 4);
        polyhedronsList.put("Cube", 6);
        polyhedronsList.put("Octahedron", 8);
        polyhedronsList.put("Dodecahedron", 12);
        polyhedronsList.put("Icosahedron", 20);

        int numberOfPolyhedrons, totalFaces = 0;
        numberOfPolyhedrons = scanner.nextInt();
        for (int i = 1; i <= numberOfPolyhedrons; i += 1) {
            String nameOfPolyhedron = scanner.next();
            totalFaces += polyhedronsList.get(nameOfPolyhedron);
        }
        System.out.println(totalFaces);
        scanner.close();
    }
}

File: AntonAndPolyhedrons.cpp
#include <iostream>
#include <string>
#include <unordered_map>
using namespace std;

int main() {
    unordered_map<string, int> polyhedronsList = {
        {"Tetrahedron", 4},
        {"Cube", 6},
        {"Octahedron", 8},
        {"Dodecahedron", 12},
        {"Icosahedron", 20}
    };
    int numberOfPolyhedrons, totalFaces = 0;
    cin >> numberOfPolyhedrons;
    for (int i = 1; i <= numberOfPolyhedrons; i++) {
        string nameOfPolyhedron;
        cin >> nameOfPolyhedron;
        totalFaces += polyhedronsList[nameOfPolyhedron];
    }
    cout << totalFaces << endl;
    return 0;
}
